params/protocol_params.go

```diff
@@ -108,6 +108,10 @@ const (
 	// Introduced in Tangerine Whistle (Eip 150)
 	CreateBySelfdestructGas uint64 = 25000
 
+	BaseFeeChangeDenominator = 8          // Bounds the amount the base fee can change between blocks.
+	ElasticityMultiplier     = 2          // Bounds the maximum gas limit an EIP-1559 block may have.
```

Comment:
This constant is also only applied during the verification, but it should also be used for bumping the gaslimit in the miner package for making new block. https://github.com/ethereum/go-ethereum/blob/e43ac53264429e430450fc1054ae294900172e0d/miner/worker.go#L756

Reply:
Also there are some recommit interruption mechanism in the miner. Seems now the `GasUsed` can larger than `Gaslimit`, this logic should also be fixed.

https://github.com/ethereum/go-ethereum/blob/e43ac53264429e430450fc1054ae294900172e0d/miner/worker.go#L769

Reply:
Yep, I haven't actually done anything to the miner in this PR. Is that something you'd like also included here? I'm currently tracking those changes in a [different branch](https://github.com/quilt/go-ethereum/pull/36), @adietrichs work on it a while ago and I let it get outdated - need to rebase against the latest.

Reply:
Yup, let's leave it for now. Miner part still needs to be reworked a bit.